src/main/resources/templates/ama/view.html

```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
-<head></head>
+<head>
```

Comment:
Why is the javascript included here? The head in these fragments gets thrown away anyhow